opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -338,14 +333,13 @@ namespace Opm {
                 // chopping of the update.
                 updateState(x,reservoir_state);
                 wellModel().updateWellState(xw, well_state);
+                // if the solution is updated the solution needs to be comunicated to ebos
+                // and the cachedIntensiveQuantities needs to be updated.
+                convertInput( iteration, reservoir_state, ebosSimulator_ );
+                ebosSimulator_.model().invalidateIntensiveQuantitiesCache(/*timeIdx=*/0);
```

Comment:
Not sure how the timeIdx parameter is supposed to be used here? Should it not be the current timestep number?